aclk/schema-wrappers/node_creation.cc

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "../aclk-schemas/proto/nodeinstance/create/v1/creation.pb.h"
+#include "node_creation.h"
+
+#include <stdlib.h>
+
+char *generate_node_instance_creation(size_t *len, const node_instance_creation_t *data)
+{
+    nodeinstance::create::v1::CreateNodeInstance msg;
+
+    if (data->claim_id)
+        msg.set_claim_id(data->claim_id);
+    msg.set_machine_guid(data->machine_guid);
+    msg.set_hostname(data->hostname);
+    msg.set_hops(data->hops);
+
+    *len = msg.ByteSizeLong();
+    char *bin = (char*)malloc(*len);
```

Comment:
Considering that here we are having a situation that can create an exception, why are we not writing it inside an `try ... catch` block?

Reply:
malloc doesn't throw exceptions